client/components/fields/editor/Days.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import {superdeskApi} from '../../../superdeskApi';
+import {WEEK_DAY, IEditorFieldProps} from '../../../interfaces';
+
+import {EditorFieldCheckboxGroup} from './base/checkButtonGroup';
+
+interface IProps extends IEditorFieldProps {
+    defaultAllOn?: boolean;
+}
+
+export class EditorFieldDays extends React.PureComponent<IProps> {
+    render() {
+        const {gettext} = superdeskApi.localization;
+
+        return (
+            <EditorFieldCheckboxGroup
+                field={this.props.field ?? 'week_days'}
+                label={this.props.label ?? gettext('Week Days')}
+                options={[{
```

Comment:
I think we will need some config for starting day of the week, maybe we even have it somewhere